src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
```

Comment:
Ensure that the image path `../images/star.svg` is correct and the image exists in the specified directory.